src/pages/TabsPage.tsx

```diff
@@ -0,0 +1,46 @@
+import { Link, useParams } from 'react-router-dom';
+import { Tab } from '../types/Tab';
+
+type Props = {
+  tabs: Tab[];
+};
+
+export const TabsPage: React.FC<Props> = ({ tabs }) => {
+  const { tabId = 0 } = useParams();
+
+  const selectedTab = tabs.find(tab => tab.id === tabId);
+
+  return (
+    <>
+      <h1 className="title">Tabs page</h1>
+
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              data-cy="Tab"
+              key={tab.id}
+              className={tab.id === tabId ? 'is-active' : ''}
+            >
+              <Link to={`../${tab.id}`}>
+                {tab.title}
+              </Link>
+            </li>
+          ))}
+        </ul>
+      </div>
+
+      {selectedTab
+        ? (
+          <div className="block" data-cy="TabContent">
+            {selectedTab?.content}
+          </div>
+        )
+        : (
+          <div className="block" data-cy="TabContent">
+            Please select a tab
+          </div>
+        )}
```

Comment:
You can use the wrapper once and choose the content inside

```suggestion
    <div className="block" data-cy="TabContent">
      {selectedTab
        ? selectedTab?.content
        : 'Please select a tab'
       }
    </div>
```